### PR TITLE
fix(accessibility): Changed colors of links and operator header copy

### PR DESCRIPTION
**Pull request by chrisdemars**
I changed the colors of the links to a darker blue and the operator header copy to white to fix color contrast errors.

### CODE REVIEW

File: src/app/operators/_operator-theme.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 $operator-border: #f3f3f3;
 $operator-active: #62757f;
-$link-color: #2196f3;
+$link-color:#0a6fc2;
```
Comment (guilhermejcgois) [inline src/app/operators/_operator-theme.scss]: Not a big deal, but isn't better to keep the space between colon and #? Just a cosmetic point

**Comment by chrisdemars** [comment]
Ooooh that is my fault. I thought I put the space in, sorry.

**Comment by guilhermejcgois** [comment]
> Ooooh that is my fault. I thought I put the space in, sorry.

Glad to help

**Comment by chrisdemars** [comment]
I put that fix in @guilhermejcgois

**Comment by ashwin-sureshkumar** [comment]
@chrisdemars - Nice job. Can you please squash your commits and push ? Thank you !

**Comment by chrisdemars** [comment]
I don't know how to squash commits @ashwin-sureshkumar. I have never had to do that before.

**Comment by chrisdemars** [comment]
Thanks for the merge

**Pull request merged**